src/Network/Network.Test/ScenarioTests/NetworkWatcherAPITests.ps1

```diff
@@ -108,6 +108,20 @@ function Get-NrpResourceGroupName
 	Get-ResourceGroupName "psnrp";
 }
 
+function Wait-Vm($vm)
+{
+    # Don't wait more than N minutes to avoid getting stuck in a loop if VM can't recover
+    $minutes = 30;
+    while((Get-AzVM -ResourceGroupName $vm.ResourceGroupName -Name $vm.Name).ProvisioningState -ne "Succeeded")
+    {
+        Start-Sleep 60;
```

Comment:
This will make all tests wait for this long in playback as well as record mode.  Instead use the `Start-TestSleep` function defines here: https://github.com/Azure/azure-powershell/blob/master/src/Network/Network.Test/ScenarioTests/Common.ps1#L98  or the `Wait-Seconds` cmdlet defined here: https://github.com/Azure/azure-powershell/blob/master/tools/ScenarioTest.ResourceManager/Common.ps1#L307